src/Skoruba.IdentityServer4.Admin/Controllers/IdentityController.cs

```diff
@@ -1,18 +1,22 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Text.Encodings.Web;
```

Comment:
In this file.. There are some introductions of the ability to send e-mails similiar to the STS.Identity project.